src/app/header/header.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent implements OnInit {
+
+  LOGO = require("../images/icon.png"); // doesn't show image without 'require'

```

Comment:
It's not the proper way of deal with images. Please, either add image via css style(using background-url property) or put it as <img> tag with full path from assets directory